components/camera/transformpipeline/mods.go

```diff
@@ -158,16 +160,19 @@ func (rs *resizeSource) Close(ctx context.Context) error {
 
 // cropConfig are the attributes for a crop transform.
 type cropConfig struct {
-	XMin int `json:"x_min_px"`
-	YMin int `json:"y_min_px"`
-	XMax int `json:"x_max_px"`
-	YMax int `json:"y_max_px"`
+	XMin        float64 `json:"x_min_px"`
+	YMin        float64 `json:"y_min_px"`
+	XMax        float64 `json:"x_max_px"`
+	YMax        float64 `json:"y_max_px"`
+	ShowCropBox bool    `json:"overlay_crop_box"`
 }
 
 type cropSource struct {
 	originalStream gostream.VideoStream
 	imgType        camera.ImageType
 	cropWindow     image.Rectangle
+	cropRel        []float64
```

Comment:
Part of me says "make this a struct with 4 fields, so it's obvious which one is XMin, which one is YMin, etc." Part of me says "leave this as a list, because it's easier to write and creating a struct isn't worth the hassle." Just something to consider...